packages/cli/src/CLI.ts

```diff
@@ -59,7 +59,8 @@ export class CLI implements Command {
     }
 
     const cmd = this.cmds[cmdName]
-    if (cmd) {
+    const isVisible = cmdName !== 'platform' || Boolean(args['--early-access-feature'])
+    if (cmd && isVisible) {
```

Comment:
You can remove this here and handle it in the platform command file, it would be better I think

Reply:
Check https://github.com/prisma/prisma/pull/19447 for how to do it, see “packages/migrate/src/commands/MigrateCommand.ts” file

Reply:
I made a change before seeing your second comment. I think the current approach does the job though well enough?